im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,425 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Im Gespräch - Hefte der Martin Buber-Gesellschaft (German)</title>
+    <title-short>Im Gespräch</title-short>
+    <id>http://www.zotero.org/styles/im-gespraech-buber-gesellschaft</id>
```

Comment:
So it should be;

  <info>
    <title>Im Gespräch - Hefte der Martin Buber-Gesellschaft (German)</title>
    <title-short>Im Gespräch</title-short>
    <id>http://www.zotero.org/styles/im-gesprach</id>
    <link href="http://www.zotero.org/styles/im-gesprach" rel="self"/> 

correct?